src/dmd/globals.h

```diff
@@ -295,18 +295,30 @@ struct DArray
     T *ptr;
 };
 
+enum FilenameType
```

Comment:
why not C++11 enum class?
```c++
enum class FilenameType{
  default_, pound_line_, mixin_
};
```

Reply:
I don't see this feature used anywhere else in the code base.